content_scripts/hints.js

```diff
@@ -363,6 +364,8 @@ var Hints = (function() {
                     }
                 });
                 elements = filterOverlapElements(elements);
+                var scrollables = Normal.refreshScrollableElements();
```

Comment:
Thanks for the PR.

`f` is such a common command, to put this line here may hurt user experience as it may double the time to display hints.

I like the separate command `;fs` to show hints for scrollable elements. Could you also test the change with Firefox?

Reply:
removed it from the 'f' command for performance consideration; tested it on firefox.